content/blog/2018-06-26-roomba.Rmd

```diff
@@ -0,0 +1,107 @@
+---

```

Comment:
Can you please add this preface to YAML? All unconf18 project posts will have same, to distinguish these from posts about reviewed and onboarded packages. Example use is here: https://raw.githubusercontent.com/ropensci/roweb2/master/content/blog/2018-06-08-rprofile-julia-silge.md
https://ropensci.org/blog/2018/06/08/rprofile-julia-silge/

`preface: "This post describes a project from rOpenSci unconf18. In the spirit of exploration and experimentation at our unconferences, projects are not necessarily finished products or in scope for rOpenSci packages."`

Reply:
Done!